src/page/Mypage/MystorePage/index.tsx

```diff
@@ -0,0 +1,72 @@
+import { ReactComponent as EventMarkIcon } from 'assets/svg/mystore/event-menu-mark.svg';
+import useModalOpen from 'utils/hooks/useModalOpen';
+import styles from './MystorePage.module.scss';
+import Menus from './Menus';
+import AddMenuModal from './AddMenuModal';
+
+const storeInfos = ['전화번호', '운영 시간', '휴무일', '주소정보', '배달 금액', '가게 정보'];
+
+export default function MystorePage() {
+  const { isOpenModal, clickModal, closeModal } = useModalOpen();
+
+  return (
+    <div className={styles.container}>
+      <div className={styles.section}>
+        <div className={styles.header}>
+          <div className={styles.header__title}>가게정보</div>
+          <div className={styles['header__btn-wrapper']}>
+            <button type="button" className={styles['header__btn-update']}>메뉴수정</button>
+            <button
+              type="button"
+              className={styles['header__btn-add']}
+              onClick={clickModal}
+            >
+              메뉴추가
+            </button>
+          </div>
+        </div>
+        <div className={styles.store}>
+          <div className={styles.store__info}>
+            <div className={styles.store__top}>
+              <div className={styles.store__title}>가장 맛있는 족발</div>
```

Comment:
조금은 의미론적인 태그들을 사용해주세요 ~~ 
```suggestion
              <h1 className={styles.store__title}>가장 맛있는 족발</h1>
```

Reply:
네! 해당 부분과 관련 부분 수정 했습니다.
앞으로 진행 할 부분에서 참고하겠습니다!